### PR TITLE
Add Marker class, Scenarios enum

### PR DESCRIPTION
**Pull request by Rich-Dunne**
Implemented Scenarios enum into StartScenarioInPlace wrapper method and TaskStartScenarioInPlace extension method.

### CODE REVIEW

**Comment by Rich-Dunne** [comment]
Accidentally did not include Marker changes from other branch in this one.  Updated to reflect changes in other branch.  Should only have to focus on PedExtensions, Scenarios, and NativeTaskWrappers

**Comment by NightYield** [comment]
Why is the marker included at all? Without this dependency the branch would have been standalone.

**Comment by Rich-Dunne** [comment]
> 
> 
> Why is the marker included at all? Without this dependency the branch would have been standalone.

It was an accident.

File: Source/RageCommunity.Library/Extensions/PedExtensions.cs [hunk 1]
```diff
@@ -27,7 +28,7 @@ public static class PedExtensions
         /// <remarks>
         /// A list of scenarios can be found <a href="https://github.com/DurtyFree/gta-v-data-dumps/blob/master/scenariosCompact.json">here</a>.
         /// </remarks>
-        public static void StartScenarioInPlace(this Ped ped, string scenarioName, int unkDelay = 0, bool playEnterAnimation = true) => NativeWrappers.TaskStartScenarioInPlace(ped, scenarioName, unkDelay, playEnterAnimation);
+        public static void StartScenarioInPlace(this Ped ped, Scenarios scenarioName, int unkDelay = 0, bool playEnterAnimation = true) => NativeWrappers.TaskStartScenarioInPlace(ped, scenarioName, unkDelay, playEnterAnimation);
```
Comment (NightYield) [inline Source/RageCommunity.Library/Extensions/PedExtensions.cs]: Why is the parameter called scenarionName when the type was changed from string to Scenarios?

**Review by NightYield — CHANGES_REQUESTED** [review]

File: Source/RageCommunity.Library/Extensions/PedExtensions.cs (hunk 1 above)
Comment (Rich-Dunne) [reply Source/RageCommunity.Library/Extensions/PedExtensions.cs]: Because I changed the type and not the parameter name.

**Review by NightYield — APPROVED** [review]

**Pull request merged**